packages/cli/src/commands/webhook.ts

```diff
@@ -110,6 +111,11 @@ export class Webhook extends BaseCommand {
 
 	async initOrchestration() {
 		await Container.get(OrchestrationWebhookService).init();
-		await Container.get(OrchestrationHandlerWebhookService).init();
+
+		const subscriber = Container.get(Subscriber);
+		await subscriber.subscribe('n8n.commands');
+		subscriber.setCommandMessageHandler();
+
+		Container.get(PubSubHandler).init();
```

Comment:
```suggestion
		Container.get(PubSubHandler).init();

		const subscriber = Container.get(Subscriber);
		subscriber.setCommandMessageHandler();
		await subscriber.subscribe('n8n.commands');
```

Would this order also work and prevent use form missing messages?

Reply:
Thanks! It's a tiny window - I'll address in the next PR 👍🏻